assets/src/dashboard/app/views/savedTemplates/index.js

```diff
@@ -50,6 +50,7 @@ import {
   PageHeading,
   StoryGridView,
 } from '../shared';
+import { STORY_SORT_MENU_ITEMS } from '../../../constants';
```

Comment:
Nit, looks like you're importing from `../../../constants` in two different `imports` 😱

Reply:
thank you, missed this.